devtools/gradle/src/main/java/io/quarkus/gradle/AppModelGradleResolver.java

```diff
@@ -339,7 +344,7 @@ private void collectExtensionDeps(Set<ResolvedDependency> resolvedDeps,
      * A {@link ResolvedArtifact} is valid if it's a JAR or a directory
      */
     private static boolean isDependency(ResolvedArtifact a) {
-        return BootstrapConstants.JAR.equalsIgnoreCase(a.getExtension()) ||
+        return BootstrapConstants.JAR.equalsIgnoreCase(a.getExtension()) || "exe".equalsIgnoreCase(a.getExtension()) ||
```

Comment:
Won't this include an exe file into the lib?

Reply:
Do you mean `lib` directory used in runtime?

Without it, artifacts with packaging exe were not among the dependencies.
And this is how protoc is shipped.

Reply:
I checked `integration-tests/gradle/build/resources/test/grpc-multi-module-project/application/build/lib` where the grpc code gen is used and I don't see the exe files there.
Isn't it the same as deployment jars?

Reply:
Ok